tasks/models.go

```diff
@@ -0,0 +1,17 @@
+package tasks
+
+import "fmt"
+
+type Task struct {
+	ID      string `json:"id,omitempty" db:"id,omitempty"`
+	Date    string `json:"date"         db:"date"`
+	Title   string `json:"title"        db:"title"`
+	Comment string `json:"comment"      db:"comment"`
+	Repeat  string `json:"repeat"       db:"repeat"`
+}
+
+func (task Task) String() string {
+	return fmt.Sprintf(
```

Comment:
Можно заменить на и будет то же самое
```suggestion
	return fmt.Sprintf("%+v", task)
```

https://pkg.go.dev/fmt

Reply:
Хотел поблагодарить за работу в течение курса. Спасибо за ревью, надеюсь когда-нибудь применю полученные знания )